install/kubernetes/helm/istio/values.yaml

```diff
@@ -24,6 +24,10 @@ galley:
 #
 mixer:
   enabled: true
+  istio-policy:
```

Comment:
may be skip the option above then?

Reply:
I would be fine to remove it.  I kept it in there for backwards compatible and consistent with the gateway's.